OBAKit/Helpers/OBAEmailHelper.m

```diff
@@ -165,13 +165,23 @@ + (NSArray*)debuggingInfoWithModelDAO:(OBAModelDAO*)modelDAO currentLocation:(CL
     
     if (@available(iOS 12.0, *)) {
         int cellProvidersIndex = 0;
-        for (CTCarrier* carrier in networkInfo.serviceSubscriberCellularProviders) {
-            [debuggingInfo addObject:@[[NSString stringWithFormat:@"Carrier %@ name: %@", @(cellProvidersIndex), carrier.carrierName]]];
+        for (id carrier in networkInfo.serviceSubscriberCellularProviders) {
+            CTCarrier* carrierInfo = [networkInfo.serviceSubscriberCellularProviders objectForKey:carrier];
+            NSString* stringName = [NSString stringWithFormat:@"Carrier %d Name", cellProvidersIndex];
+
+            // Cannot add a literal `nil` to the debugging info dictionary, so stringify it.
+            NSString* carrierName = @"nil";
```

Comment:
Although it isn't necessary to do it in this case, you can use C's ternary operator to collapse this into one line:

```
NSString *carrierName = carrierInfo.carrierName ?: @"nil";
```